torch_xla/csrc/ops/index_op.cpp

```diff
@@ -0,0 +1,163 @@
+#include "torch_xla/csrc/ops/index_op.h"
```

Comment:
I think we leave an empty line by style:

```
#include "THIS_CPP_H"

#include <SYSTEM_INCS>
...

#include <NON_LOCAL_INCS>
...

#include "LOCAL_INCS"
...

namespace ... {
```

Reply:
Done.